### DISCUSSION TITLE
漢化進度還有還原問題

### DISCUSSION THREAD

**Discussion by HaseoCorleone** [GpointChen/FFXIVChnTextPatch-GP#113] [General]
請問現在釋出的檔案已經有6.0的劇情嗎？
還有之前官方小更新有先還原一次，可是進遊戲看還是中文的？我可以用現在新的版本程式還原再漢化嗎？
還是只能重新下載官方資料了再漢化呢？

**Comment by GpointChen** [comment]
> 請問現在釋出的檔案已經有6.0的劇情嗎？

已更新至6.0，6.0X仍只有部分民間翻譯。

> 還有之前官方小更新有先還原一次，可是進遊戲看還是中文的？我可以用現在新的版本程式還原再漢化嗎？ 還是只能重新下載官方資料了再漢化呢？

如果還原後進遊戲看還是中文，代表你之前可能就有重複漢化過，導致backup資料夾內的檔案被漢化過的檔案覆蓋。

因此，你可能需要重新下載或向朋友要原始檔案。

**Reply by HaseoCorleone** [reply]
嗯感謝,那原始檔是那三個0a000嗎？還是之後我漢化完馬上把backup另外存下來以備不時之需呢？

**Reply by GpointChen** [reply]
是的，000000是字體相關，0a0000是翻譯相關。

**Reply by HaseoCorleone** [reply]
請問 剛剛我重新下載檔案完漢化了，不是都會自動備份，那個backup資料夾在哪邊呢？沒看到呢

**Reply by HaseoCorleone** [reply]
剛剛試著還原發現按下去馬上就好了，進遊戲後還是沒有還原呢？是不是現在不會備份還是什麼問題呢？

**Reply by GpointChen** [reply]
> 請問 剛剛我重新下載檔案完漢化了，不是都會自動備份，那個backup資料夾在哪邊呢？沒看到呢

![image](https://user-images.githubusercontent.com/34565875/159172500-a0ac7090-0a25-4648-929e-b6daa208d7be.png)

**Reply by GpointChen** [reply]
你的backup資料夾內有檔案嗎？

**Reply by HaseoCorleone** [reply]
壓縮檔內有看到backup,解完壓縮沒看到耶....

**Reply by GpointChen** [reply]
這個backup是空資料夾，如果你解壓會沒有這個資料夾，可能是你使用的解壓軟體會自動忽略空資料夾；你可以手動創建一個。

結合來看，你可能是因為沒有backup資料夾，所以在替換時沒有備份，才導致按了還原沒有還原，因為沒有檔案可供還原。

**Reply by HaseoCorleone** [reply]
看來是這樣，我換軟體就有了....XD
已經漢化下去了...來不及備份了....orz

**Reply by HaseoCorleone** [reply]
感謝您的分享與解答😭

**Reply by GpointChen** [reply]
我順便給你我的原始檔案好了。

**Reply by GpointChen** [reply]
https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.55/backup-0320.rar

**Reply by HaseoCorleone** [reply]
感激不盡😭
